### DISCUSSION TITLE
Is there autocompletion and code linting?

### DISCUSSION THREAD

**Discussion by Joe201610a** [cpeditor/cpeditor#1013] [Q&A] [answered]
I want code linting where basically there is a red line below the code that is wrong and also autocompletion like when i type ve it shows me options to choose from like vector.push_back() and stuff like that

**Comment by ouuan** [comment, marked as answer]
Linting: https://cpeditor.org/docs/setup/#setup-language-server

Auto-complete: currently no. There's an issue for it: #156.
